src/components/Nav/nav-BASE.js

```diff
@@ -1,90 +1,113 @@
-import React from 'react';
+ /* eslint-disable */
```

Comment:
This file seemed like a draft so I disabled the whole check here

Reply:
I believe it may have been a file I was working with as a "base" when I was having issues. Likely will be removed with #45 or a future issue for code cleanup